apps/mobile/src/hooks/use-pressed-state.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
```

Comment:
This is a hook so I don't think it should be a `.tsx` file?

Maybe this could be moved to the `ui` package, located alongside `Pressable`. 

Then you could also remove the large comment into a story file instead to act more as documentation. We don't have any stories for `pressable.native` now